schemas/SupplyConnection.yaml

```diff
@@ -0,0 +1,51 @@
+type: object
+description: >
+  A connection denotes a physical connection to a district heating supplier.
+  Contains the metadata regarding the sensors on the supply-side of district heating heat exchanger.
+required:
+  - id
+  - name
+  - feedTemperatureSensorId
+  - returnTemperatureSensorId
+  - energyMeterSensorId
```

Comment:
Is it likely that these values will be shared with HeatingSystem or RadiatorLoop?

Reply:
No, they can never be shared with them. It is possible that RadiatorLoop and HeatingSystem share a common sensor, but the SupplyConnection sensors are always different. SupplyConnection sensor measure things on the supply side of the heat exchanger, while RadiatorLoop and HeatingSystem measure things on the demand side.